xml/QS_system_requirements.xml

```diff
@@ -17,7 +17,127 @@
   <title>&productname; System Requirements</title>
  </info>
  <para>
-  The following system requirements apply for all types of nodes provided by
-  &productname;:
+  Before you begin the installation, please make sure your system meets all the
+  requirements listed below.
  </para>
+ <sect1 xml:id="sec.caasp.quickstart.clusreqs">
+    <title>Cluster Size Requirements</title>
+    <para>
+     &productname; only functions in a clustered configuration. It requires a 
+     group of at least four physical or virtual machines. This will yield a 
+     cluster with one &admin_node;, one &master_node;, and two &worker_node;s.
+    </para>
+    <para>
+     All clusters must contain a single &admin_node; and at least one &master_node;
+     and one &worker_node;.
+    </para>
+    <para>
+     For improved performance and reliability, an odd number of &master_node;s 
+     and any number of &worker_node;s up may be added.
+    </para>
+    <note>
+     <title>Test and Proof-of-Concept Clusters</title>
+     <para>
+      It is possible to provision a three-node cluster with only a single
+      &worker_node;, but this is not a supported configuration for deployment.
+     </para>
+    </note>
+   </sect1>
+   <sect1 xml:id="sec.caasp.quickstart.hwreqs">
+    <title>Minimum Hardware Requirements</title>
+    <itemizedlist mark="bullet" spacing="normal">
+     <listitem>
+      <para>
+       Each node must have an AMD64/Intel* EM64T processor; quad-core
+       processors are recommended. 32-bit processors are not supported.
+      </para>
+     </listitem>
+     <listitem>
+      <para> 8 GB physical RAM per node. </para>
+     </listitem>
+     <listitem>
+      <para>
+       40 GB available disk space (or more depending on the count and size of 
+       the container images you intend to run).
+      </para>
+     </listitem>
```

Comment:
Network card requirements are missing. Is one NIC per node sufficient? Does the admin node require more NICs?

Reply:
I will ask -- I have not seen any mention of this. It will function with 1 NIC per node as that is all I have seen.